test/__snapshots__/plugin.test.ts.snap

```diff
@@ -9,7 +9,7 @@ exports[`additional body 1`] = `
       
     </head>
     <body>
-      <div>Demo</div><script src=\\"runtime~main.js\\"></script><script src=\\"main.js\\"></script>
+      <div>Demo</div><script src=\\"main.js\\"></script>
```

Comment:
Are these changes intentional?

Reply:
Yes, they have to do with webpack 5 default behavior as there the runtime is included to the output bundles by default.